### DISCUSSION TITLE
V7.0 is OUT

### DISCUSSION THREAD

**Discussion by arkypita** [arkypita/LaserGRBL#2259] [General]
New features: https://github.com/arkypita/LaserGRBL/releases/tag/v7.0.0
Please report here your impressions and any issues with v7.0.0.

**Comment by Papas1985** [comment]
hello after i install the new version 7.0 i am getting crash app closing before open ! any info

**Reply by arkypita** [reply]
LaserGRBL session log is in C:\Users\YOUR-ACCOUNT\AppData\Roaming\LaserGRBL.
The file is called "sessionlog.txt" could you please open and see if it show any errors at the end?

![image](https://github.com/arkypita/LaserGRBL/assets/8782035/1616c3b7-a54a-4c67-9d70-3babe2ef9115)



Also check in windows event log if is there some log related to LaserGRBL under Windows Logs, Application. Something like this:
![image](https://github.com/arkypita/LaserGRBL/assets/8782035/2211769a-c8b0-46cf-8c55-5458f75bfbc1)




Let me know

**Reply by arkypita** [reply]
Is it possible to download previous version from here: https://github.com/arkypita/LaserGRBL/releases/download/v6.2.2/install.exe

**Comment by rotkop456** [comment]
Hello @arkypita, Trust you are well. I have a TTS-55 Pro with the new updates from 7 to 7.7 I have been struggling to do a cut out on 3mm MDF wood with my laser. It burns the wood as you can see in the image, Will you be able to point me in the right direction to fix this?

I also want to find out if there is a way to reset the software to default settings?

![Cut test from LaserGRBL](https://github.com/arkypita/LaserGRBL/assets/9168743/5659c256-ceb5-4f47-9f0f-2fc57aeaa0f0)

**Reply by arkypita** [reply]
Hi @rotkop456

The changes between versions 6.x and 7.x up to 7.7 concern only the graphic interface: a restyling of the icons and a preview window that allows you to zoom and see the progress of the work in real time. 

Nothing has changed in how LaserGRBL works and the parameters to use. Even more so the most important parameters are actually managed by the laser engraver, and reside on its control board, so there really is nothing in the updates that changes how the laser behaves.

Looking at the images you uploaded it seems clear that the laser is out of focus, an out of focus laser will not cut but will leave very large burns.
